infra/test/integration/simple_example/simple_example_test.go

```diff
@@ -58,6 +58,39 @@ func TestSimpleExample(t *testing.T) {
 			return false, nil
 		}
 		utils.Poll(t, isServing, 10, time.Minute*1)
+
+		// Check cloud run jobs status
+		isJobCompletion := func() (bool, error) {
+			cloudRunJobs := gcloud.Run(t, ("beta run jobs list --format=json"), gcloudArgs).Array()
+			var jobResults []bool
+			for _, cloudRunJob := range cloudRunJobs {
+				jobName := cloudRunJob.Get("metadata.name").String()
+				jobCompletionTime := cloudRunJob.Get("status.latestCreatedExecution.completionTimestamp").String()
+				jobExecutionCount := cloudRunJob.Get("status.executionCount").Int()
+				fmt.Printf("Job %s completion time: %s, executionCount: %d\n", jobName, jobCompletionTime, jobExecutionCount)
+				if jobCompletionTime == "" || jobExecutionCount < 1 {
+					fmt.Printf("Job %s is not completed\n", jobName)
+					jobResults = append(jobResults, false)
+				} else {
```

Comment:
nit: I would rewrite this without the `else` statement.

```
if jobCompletionTime == "" || jobExecutionCount < 1 {
	fmt.Printf("Job %s is not completed\n", jobName)
	jobResults = append(jobResults, false)
        continue
}

fmt.Printf("Job %s is completed\n", jobName)
jobResults = append(jobResults, true)
```

Reply:
fixed in [12c25e4](https://github.com/GoogleCloudPlatform/terraform-large-data-sharing-golang-webapp/pull/19/commits/12c25e45d7043e134d12b0d5b91dd4002c41d660)